### DISCUSSION TITLE
📣 VSCode users, please upgrade to version 1.20

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#16815] [Copilot Conversations]
VScode pilots 👋 older versions of the Copilot extension are no longer supported. Please upgrade to version 1.20 or later.

**Comment by iamvucms** [comment]
```[INFO] [fetch] [2022-05-18T07:51:42.189Z] Please upgrade your Copilot extension to continue using this service.```
I'm using vscode 1.67.1 and latest copilot version. Above issue prevent me to use copilot

**Reply by Butanium** [reply]
What is your VS-code version ? I had this error because my VScode wasn't updated

**Reply by iamvucms** [reply]
Ah, just seen "Reload require" after upgrade to latest vscode version. Copilot now worked
